cmd/minikube/cmd/config/profile_list.go

```diff
@@ -58,8 +59,18 @@ var profileListCmd = &cobra.Command{
 	},
 }
 
+func listProfiles() (validProfiles, invalidProfiles []*config.Profile, err error) {
+	if isLight {
+		validProfiles, err = config.ListValidProfiles()
```

Comment:
lets rename ListValidProfiles to something else, ``ListProfilesNoVerify()` or `ListProfilesLigth()`

Reply:
@priyawadhwa disagreed to rename to `ListProfilesLigth`  https://github.com/kubernetes/minikube/pull/10119#discussion_r565568934 

`ListProfilesNoVerify()` is not suitable.
I verified profile to ensure Config struct is not empty.
https://github.com/kubernetes/minikube/blob/ce8a7e8014373193d69ec9be7bc16a85ab70e28a/pkg/minikube/config/profile.go#L243-L244
Also, i think `ListValidProfiles` will not miss any valid profile. Although, we can find more profile through Docker in `ListProfile` func, `LoadProfile` will be failed if config file not existing in minihome. so only different between two functions is infos from container(status)

Reply:
sounds good !